packages/app/src/components/admin/AdminPage.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useCallback, useContext, useEffect } from 'react';
+import {
+  Content,
+  ErrorPage,
+  Header,
+  HeaderTabs,
+  Page,
+} from '@backstage/core-components';
+import DynamicRootContext from '../DynamicRoot/DynamicRootContext';
+import Grid from '@mui/material/Grid';
+import Box from '@mui/material/Box';
+import { Link, useLocation, useNavigate } from 'react-router-dom';
+
+const tabs = [
+  {
+    id: 'rbac',
+    label: 'RBAC',
+    tabProps: { to: '/admin/rbac', component: Link },
+  },
+  {
+    id: 'plugins',
+    label: 'Plugins',
+    tabProps: { to: `/admin/plugins`, component: Link },
+  },
+];
+
+export const AdminPage = () => {
```

Comment:
good to add some unit tests for this. Can be taken up as follow up PR if needed

Reply:
yep for sure

Reply:
https://github.com/janus-idp/backstage-showcase/issues/940 unless I finish it before this PR is merged.

Reply:
Added https://github.com/janus-idp/backstage-showcase/pull/932/files#diff-43884aaa2324a9c65904cce4a0137a8754deb314ccdd58112fb9a31a13bb56a9